vispy/app/qt.py

```diff
@@ -17,11 +17,16 @@
 
 
 if qt_lib in ('pyqt4', 'pyside'):
+    # Why is this not a seperate option for pyside.
+    # PySide and PyQt4 are different aren't they?
```

Comment:
Yes this does seem strange doesn't it. If @larsoner or @kmuehlbauer don't have any ideas why this is like this, it seems like it should be split in to a separate if branch.

Reply:
No idea, seems odd to me too.

Reply:
Agreed this looks like a bug